language/types_values_variables.md

```diff
@@ -142,16 +146,17 @@ may appear more than once in the hierarchy (e.g. a `Scalar` is both `Any` and `D
        |  |     |- (Integer with range inside another Integer)
        |  |  |- Float[from, to]
        |  |     |- (Float with range inside another Float)
+       |  |  |- Timespan
+       |  |  |- Timestamp
        |  |
        |  |- String[from, to]
        |  |  |- Enum[*strings]
        |  |  |- Pattern[*patterns]
        |  |
        |  |- Boolean
        |  |- Regexp[pattern_string]
-       |
-       |- SemVer
-       |- SemVerRange
+       |  |- SemVer
+       |  |- SemVerRange
```

Comment:
It is a bug that Regexp is Data. It has not been fixed yet because it is a breaking change. Not sure we have a ticket for that.